pkg/kv/kvserver/rangelog/rangelog.go

```diff
@@ -86,22 +80,11 @@ func (s *Writer) WriteRangeLogEvent(
 		}
 		args[5] = tree.NewDString(string(infoBytes))
 	}
-	entries, err := rowenc.EncodePrimaryIndex(
-		s.codec,
-		s.tableDesc,
-		s.primaryIndex,
-		s.tableColMap,
-		args[:],
-		false, // includeEmpty
-	)
-	if err != nil {
+	ba := txn.NewBatch()
+	if err := s.w.Insert(ba, args[:]...); err != nil {
 		return errors.NewAssertionErrorWithWrappedErrf(
 			err, "failed to encode rangelog index entries",
 		)
 	}
-	ba := txn.NewBatch()
-	for i := range entries {
-		ba.Put(entries[i].Key, &entries[i].Value)
```

Comment:
I assume replacing this `Put` with the CPut-variant in `Insert` is OK?

Reply:
Correct, the CPut is better.